LibVLCSharp.Gtk.Sample/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Gtk;
+using LibVLCSharp.Platforms.Gtk;
+using LibVLCSharp.Shared;
+using System.Runtime.InteropServices;
+
+namespace LibVLCSharp.Gtk.Sample
+{
+    using System;
+
+    public class Program
+    {
+        public static void Main()
+        {
+            // Initializes X threads before calling VLC. This is required for vlc plugins like the VDPAU hardware acceleration plugin.
+            if(RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                NativeReferences.XInitThreads();
```

Comment:
`This function returns a nonzero status if initialization was successful; otherwise, it returns zero.`
Should we check for a return value here? And output a warning if that failed?

Reply:
Yes, we should